tests/lib/cli-engine.js

```diff
@@ -1536,10 +1537,20 @@ describe("CLIEngine", function() {
                 }
             }
 
+            /**
+             * helper method to delete the cache files created during testing
+             * @returns {void}
+             */
+            function delCache() {
```

Comment:
Maybe a nitpick, but could we call this `deleteCache` instead? There's no need to abbreviate `delete` to `del` here.